src/components/guide/index.js

```diff
@@ -0,0 +1,221 @@
+import PropTypes from "prop-types";
+import TraitifyPropTypes from "lib/helpers/prop-types";
+import {Component} from "react";
+import withTraitify from "lib/with-traitify";
+import guideQuery from "lib/graphql/queries/guide";
+import {dangerousProps, listId} from "lib/helpers";
+import smallScreen from "./helpers/helpers";
+import style from "./style";
+
+class Guide extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      badges: [],
+      competencies: [],
+      displayedCompetency: {},
+      errors: [],
+      expandedIntro: false
+    };
+  }
+  static defaultProps = {assessment: null};
+  static propTypes = {
+    airbrake: PropTypes.shape({notify: PropTypes.func}).isRequired,
+    assessment: PropTypes.shape({personality_types: PropTypes.array}),
+    assessmentID: PropTypes.string.isRequired,
+    locale: PropTypes.string.isRequired,
+    translate: PropTypes.func.isRequired,
+    traitify: TraitifyPropTypes.traitify.isRequired
+  }
+  componentDidMount() {
+    this.setGuide();
+  }
+  componentDidUpdate(prevProps) {
+    if(this.props.locale !== prevProps.locale) {
```

Comment:
I think we'd also need to reload it if the assessment ID changed

Reply:
I believe the assessmentID only was getting updated when an assessment change occurred. I added something for this, but open to suggestions on it.